packages/cxl-lumo-styles/scss/themes/cxl-tabs-slider.scss

```diff
@@ -125,7 +125,8 @@
   margin-top: calc(var(--lumo-space-s) * -1);
 
   ::slotted(vaadin-tab) {
-    padding: 0 calc(var(--lumo-space-s) / 2) calc(var(--lumo-space-s) / 2);
+    width: 267px;
```

Comment:
I'm a bit nervous about static widths here. Do we have rational or reasoning behind choosing this size? Different client sizes, font scaling could provide a different experience.

Reply:
@freudFlintstone @anoblet how did it work before? How card width was calculated? For me the weirdest thing is uneven size.

Reply:
@pawelkmpt 267px is size set in the [figma spec](https://www.figma.com/file/RTj2PtUSrCOUFtljOiq0W3/CXL-2.0?type=design&node-id=268-28470&mode=dev). I'm happy to change it to 270, 300, but I guess @heshfekry should ahve a say in it.

Reply:
To avoid setting a static width, we might use a % of the container + a min-width, maybe. But it would be pretty hard to avoid the different card widths depending on content without having a static value